README.rst

```diff
@@ -1,47 +1,30 @@
 Ray Tutorial
 ============
 
-Setup
------
+To run the tutuorial online, click on the binder link below:
 
-1. Make sure you have Python installed (we recommend using the `Anaconda Python
-   distribution`_). Ray works with both Python 2 and Python 3. If you are unsure
-   which to use, then use Python 3.
+.. image:: https://mybinder.org/badge.svg :target: https://mybinder.org/v2/gh/jzf2101/tutorial/master
 
-2. **Install Jupyter** with ``pip install jupyter``. Verify that you can start
-   a Jupyter notebook with the command ``jupyter-notebook``.
 
-3. **Install Ray** by running ``pip install ray``. Verify that you can run
+Local Setup
+-----------
 
-    .. code-block:: bash
+To install the tutorial on your own machine:
 
-      import ray
-      ray.init()
+1. Make sure you have installed
```

Comment:
I'd prefer not to introduce a dependency on Anaconda, since I think that will limit the audience.